synapse/config/server.py

```diff
@@ -155,9 +155,10 @@ def default_config(self, server_name, **kwargs):
             # The port to listen for HTTPS requests on.
             port: %(bind_port)s
 
-            # Local interface to listen on.
-            # The empty string will cause synapse to listen on all interfaces.
-            bind_address: ''
+            # Local addresses to listen on.
+            # This will listen on all IPv4 addresses by default.
```

Comment:
it's going to be such a common scenario (plus we want to advertise that ipv6 is here, yay!) that i suspect it's worth spelling it out (assuming you are sure that the second bind will fail?)  
how about:

```
 # uncomment to listen on all IPv6 interfaces
 # N.B. on Linux this also listens on IPv4, so you'll need to comment out the line above.
 # - '::'
```